general-concepts/licenses/text.xml

```diff
@@ -19,6 +39,86 @@ be used) then use the following syntax:
 LICENSE="|| ( foo bar )"
 </codesample>
 
+<subsection>
+<title>Determining the correct license</title>
+<body>
+
+<warning>
+Do not trust the license indicated by GitHub or other sites using
+automation to guess the license! The information presented is rarely
+complete, and may be incorrect.
+</warning>
+
+<p>
+To establish the correct value of <c>LICENSE</c>, you need to trace
+the licenses of all installed files. Normally, the licenses of output
+files (compiled executables, generated files) are implied
+by the licenses of the respective input files.
+</p>
+
+<p>
+When looking for license information, the following should be
+considered:
+</p>
+
+<ol>
+	<li>
+		<c>COPYING*</c> and <c>LICENSE*</c> files distributed with
+		the package
+	</li>
+	<li>
+		explicit statements in documentation
+	</li>
+	<li>
+		explicit copyright notices in source and data files
```

Comment:
That should be "license notices". (Sorry, I've missed this in the previous iteration.)

Reply:
Updated.